core/src/Streamly/Internal/Data/Stream/Serial.hs

```diff
@@ -71,15 +72,15 @@ type Serial = SerialT IO
 -- Generate an infinite stream by repeating a pure value.
 --
 {-# INLINE_NORMAL repeat #-}
-repeat :: Monad m => a -> SerialT m a
+repeat :: Monad m => a -> Stream m a
```

Comment:
This is the SerialT module, there is no point of changing SerialT to Stream in this.